modules/xmpp/TraceablePeerConnection.js

```diff
@@ -547,7 +547,9 @@ TraceablePeerConnection.prototype.createAnswer
                         dumpSDP(answer));
                 }
 
-                if (!this.session.room.options.disableRtx && !RTCBrowserType.isFirefox()) {
+                if (typeof this.session.room.options.disableRtx !== 'undefined'
```

Comment:
so, if i'm reading this right, this is changing "disableRtx not defined" to be treated the same as "disableRtx: true", which i think is inconsistent with the way other values are treated in the config.  if we're gonna do this, i think it would be good to go all the way through with your other idea (change 'disableRtx' to 'useRtx'), which i think is a good idea anyway.